subgraphs/limitorder/src/helper.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable prefer-const */
+import { BigInt, BigDecimal, ethereum, Address, Bytes } from '@graphprotocol/graph-ts'
+import { ERC20 } from '../generated/LimitOrder/ERC20'
+import { ERC20Bytes } from '../generated/LimitOrder/ERC20Bytes'
+import { TradedToken, User } from '../generated/schema'
+
+export let ZERO = BigInt.fromI32(0)
+export let ZERO_BD = BigDecimal.fromString('0')
+export let ONE = BigInt.fromI32(1)
+export let ONE_BD = BigDecimal.fromString('1')
+export let ZERO_ADDRESS = Address.fromString('0x0000000000000000000000000000000000000000')
+export let ETH_ADDRESS = Address.fromString('0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE')
+export let WETH_ADDRESS = Address.fromString('0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2')
+
+export let isETH = (assetAddr: Address): boolean => {
+  return assetAddr == ZERO_ADDRESS || assetAddr == ETH_ADDRESS || assetAddr == WETH_ADDRESS
+}
+
+export const addTradedToken = (tokenAddrBytes: Bytes, startDate: BigInt): TradedToken | null => {
+  const tokenAddr: Address = Address.fromBytes(tokenAddrBytes)
+  let tokenAddrStr = tokenAddr.toHex()
+  if (isETH(tokenAddr)) {
+    tokenAddrStr = WETH_ADDRESS.toHex()
```

Comment:
A bug has been identified here.
When collecting TradedToken Entity, if the assetAddr is detected as of type WETH, it will be replaced with the Mainnet WETH address ("0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2") universally.
It should instead replace it with the WETH address on the respective network.
This issue will be addressed in the next commit.